chord_metadata_service/experiments/schemas.py

```diff
@@ -84,27 +85,36 @@
             "type": "string"
         },
         "study_type": {
-            "type": "string"
+            "type": "string",
+            "enum": ["Genomics", "Epigenomics", "Metagenomics", "Transcriptomics", "Other"]
```

Comment:
I am wondering about the capitalization here. Is it of no importance or is this schema enforcing the capitalization for this property?